### PR TITLE
Prepare release: v1.0.0-alpha.8

### PR DESCRIPTION
**Pull request by tarrencev**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

## Summary by CodeRabbit

- **Version Updates**
	- Updated the software version to 1.0.0-alpha.8, indicating incremental improvements and bug fixes since the previous release.

### CODE REVIEW

**Review by kariy — APPROVED** [review]

**Comment by glihm** [comment]
Before merging, waiting feedback from for hot fixes:
- [ ] @kariy 
- [ ] @Larkooo

File: Cargo.toml
Comment (glihm) [inline Cargo.toml:54]: Adding a comment to ensure we don't merge before being ready.

**Review by glihm — CHANGES_REQUESTED** [review]

**Comment by glihm** [comment]
Will be merged once #2351 and #2353 are merged.

**Review by glihm — APPROVED** [review]

**Pull request merged**
